### PR TITLE
pkg/utils: add CPU statistics

### PR DESCRIPTION
**Pull request by holys**
Extract from tidb-enterprise-tools repo.

### CODE REVIEW

File: pkg/utils/cpu.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	"syscall"
+	"time"
+)
+
+var (
+	lastInspectUnixNano int64
```
Comment (csuzhangxc) [inline pkg/utils/cpu.go:9]: should we need `init` `lastInspectUnixNano` when process started?

Comment (holys) [reply pkg/utils/cpu.go:9]: It seems no need.   `lastCPUUsageTime` is also `0`.

**Comment by csuzhangxc** [comment]
LGTM

File: pkg/utils/cpu.go (hunk 1 above)
Comment (IANTHEREAL) [reply pkg/utils/cpu.go:9]: if we are serious, we should initial them, otherwise first `CPU usage` statistical is useless, but not matter

**Comment by IANTHEREAL** [comment]
LGTM

**Pull request merged**